tools/bin/ariaAOIassist.py

```diff
@@ -67,12 +69,14 @@ class SentinelMetadata:
         parameters and flags are added without removing metadata entries.
     '''
     ## Load data from csv and pre-format
-    def __init__(self,imgfile,track,workdir='./',excludeDates=None):
+    def __init__(self,imgfile,track,workdir='./',excludeDates=None,verbose=False):
```

Comment:
Codacy found an issue: [Missing docstring in __init__ (D107)](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=5909359)

Reply:
Hi @dbekaert, not sure I understand what this means. Could you help me fix it?
Thanks, Rob

Reply:
@rzinke it means there is no class description.

E.g. something along the lines of:
```    def __init__(self, likes_spam=False):
        """Inits SampleClass with blah."""
        self.likes_spam = likes_spam
        self.eggs = 0
```

See https://stackoverflow.com/questions/37019744/is-there-a-consensus-what-should-be-documented-in-the-classes-and-init-docst for more details